sdk/openai/openai/samples/cookbook/simpleCompletionsPage/src/index.js

```diff
@@ -0,0 +1,32 @@
+const { OpenAIClient, AzureKeyCredential } = require("@azure/openai");
+
+function getOpenAIChoices(resultDiv) {
+  const deploymentId = document.getElementById("deploymentId");
+  const azureKey = document.getElementById("azureKey");
+  const endpoint = document.getElementById("endpoint");
+  const promptInput = document.getElementById("promptInput");
+  const endpointURL = new URL(endpoint.value);
```

Comment:
the constructor takes a string.  Unless this is to validate the endpoint format?  We usually only check whether is empty in samples though.